.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -31,17 +46,27 @@ jobs:
       - name: Install dependencies
         run: pip install requests
 
+      - name: Download script file
+        # Only download the script file if the repository is not openedx/.github
+        # Because, we were not able to download correclty outside of openedx due to calling reusable workflow call
+        if: ${{ github.repository != 'openedx/.github' }}
+        run: |
+          mkdir -p scripts
+          curl -o scripts/get_ready_to_merge_prs.py https://raw.githubusercontent.com/zubairshakoorarbisoft/.github/master/scripts/get_ready_to_merge_prs.py
```

Comment:
Why is this pulling from a personal GitHub account?

Reply:
Thanks for the catch, it was left during final testing. @zubairshakoorarbisoft kindly fix that.